WalletWasabi.Fluent/Controls/TagsBox.axaml.cs

```diff
@@ -85,6 +94,12 @@ public int ItemCountLimit
 			set => SetAndRaise(CompletedCommandProperty, ref _completedCommand, value);
 		}
 
+		public ReactiveCommand<object, Unit>? DeleteTagCommand
```

Comment:
Commands should never be defined as ReactiveCommand... ICommand... take a look at Button.cs in Avalonia.